src/providers/aws.rs

```diff
@@ -188,14 +188,16 @@ impl<P> AWSLauncher<P> {
     /// instances.
     ///
     /// The lifetime of such instances must be declared in advance (1-6 hours). By default, we use 6 hours (the
-    /// maximum). If `t` > 6 hours, `AWSLauncher` will use a duration of 6 hours.
-    pub fn set_max_instance_duration(&mut self, t: i64) -> &mut Self {
+    /// maximum). If `t` > 6 hours, `AWSLauncher` will use a duration of 6 hours. If `t` == 0
+    /// hours, `AWSLauncher` will use a duration of 1 hour.
```

Comment:
I would just say "`t` will be clamped to be between 1 and 6 hours as required by EC2".